quotes/views.py

```diff
@@ -32,7 +32,8 @@ def quotes_highscore(request):
         ctx = {
             "highscore_this_semester": Quote.highscore_objects.semester_highest_score(timezone.now()),
             "highscore_all_time": Quote.highscore_objects.highest_score_all_time(),
-            "highscore_combined": combined_list # Can be used in the future so we can style the rows together
+            "highscore_combined": combined_list, # Can be used in the future so we can style the rows together
+            'pending': Quote.pending_objects.all().order_by('-created')
```

Comment:
.all() method is redundant when methods such as `order_by` or `filter`

Reply:
Fixed 💯